medusa/network_timezones.py

```diff
@@ -67,17 +67,26 @@ def update_network_dict():
     network_list = dict(cache_db_con.select('SELECT * FROM network_timezones;'))
 
     queries = []
-    for network, timezone in iteritems(d):
+    remote_network_list = iteritems(d)
```

Comment:
I still don't fully understand this change. In which situation does it get to this point? I think only if an empty json is returned. As else it would have hit line 63.

So if in some situation we want to return if no result is found. Then we should just add the check at line 65. Then we also don't need to connect to db and list all existing timezones right?

Then I would just check on `if not d: ..`. But maybe i'm missing something.